iconrpcserver/utils/json_rpc.py

```diff
@@ -94,7 +95,8 @@ def process_response(self, response, log_extra=None, log_format=None):
                     f"relay_target[{relay_target}], "
                     f"version[{version}], "
                     f"method[{method_name}]")
-        result = await CustomAiohttpClient(session, relay_uri).request(method_name, message)
+        result: Response = await CustomAiohttpClient(session, relay_uri).request(method_name, message)
```

Comment:
Is this `jsonrpcserver` Response or `jsonrpcclient` Response?

Reply:
oops. You're right. `jsonrpcclient`'s.